lib/api/model/events.dart

```diff
@@ -832,6 +834,71 @@ class UpdateMessageFlagsMessageDetail {
   Map<String, dynamic> toJson() => _$UpdateMessageFlagsMessageDetailToJson(this);
 }
 
+
+@JsonSerializable(fieldRename: FieldRename.snake)
+class TypingStatusEvent extends Event {
```

Comment:
Let's keep the event class names totally boring and predictable just from the API's `type` value: so `TypingEvent`.